okio/src/test/java/okio/DeflateKotlinTest.kt

```diff
@@ -16,6 +16,7 @@
 
 package okio
 
+import okio.ByteString.Companion.decodeHex
```

Comment:
This doesn’t seem right. Any advice? It yields the right Java API.

Reply:
<img width="332" alt="screen shot 2018-04-25 at 3 27 13 pm" src="https://user-images.githubusercontent.com/133019/39268227-3a0b6436-489d-11e8-94d8-7e51db0a3772.png">

Reply:
not yet

Reply:
Ah, yeah. We went through this on KotlinPoet. Sigh.